frontend/src/metabase/components/Calendar.unit.spec.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import moment from "moment-timezone";
+import { render, screen } from "__support__/ui";
+import MetabaseSettings from "metabase/lib/settings";
+import { updateMomentStartOfWeek } from "metabase/lib/i18n";
+import Calendar from "./Calendar";
+
+describe("Calendar", () => {
+  it("should render weekday short names", () => {
+    setup();
+
+    expect(
+      screen
+        .getAllByTestId("calendar-day-name")
+        .map((dayEl, index) => dayEl.textContent),
+    ).toEqual(["Su", "Mo", "Tu", "We", "Th", "Fr", "Sa"]);
+  });
+
+  it("should render days based on first day of the week settings", () => {
+    MetabaseSettings.set("start-of-week", "wednesday");
+    updateMomentStartOfWeek();
+
+    setup();
+
+    expect(
+      screen
+        .getAllByTestId("calendar-day-name")
+        .map((dayEl, index) => dayEl.textContent),
+    ).toEqual(["We", "Th", "Fr", "Sa", "Su", "Mo", "Tu"]);
+
+    // check that dates listed start with proper day-of-week
+    const startDate = moment().startOf("month").isoWeekday(3);
```

Comment:
Could you instead set the current date via jest, e.g.
```
jest.useFakeTimers();
jest.setSystemTime(new Date(2020, 4, 13));
```

and test for exact dates (strings)? I think it would make the test much easier to follow. For instance, I like the previous part that checks for `"We", "Th"`, it's much more straightforward.